scripts/deploy_contracts.js

```diff
@@ -26,20 +28,25 @@ async function exec() {
 
   await initWeb3()
 
+  // Retrieve deployment data for the active network.
+  try {
+    deployData = require(`../gamedata/deploy.${constants.ACTIVE_NETWORK.name}.json`)
+  }
+  catch(err){
+    deployData = {};
```

Comment:
shouldn't you halt and catch fire here? What does it mean to not have deployData?

Reply:
No, the idea is that if an object is not obtained from that address, initialize an empty object.

Reply:
Aaaaactually... if the file isn't there in the first place, it doesn't work. So I'm going to have to fix this before merge.